### DISCUSSION TITLE
Regrid atmo and ocean data from ne30pg2_EC30to60E2r2

### DISCUSSION THREAD

**Discussion by pgoddard08** [E3SM-Project/E3SM#6360] [E3SM model help]
Hi, I have output using the ne30pg2_EC30to60E2r2 grid option. I'd like to regrid the atmo and ocean data to 1x1 latlon. Do I use map_ne30np4_to_cmip6_180x360_aave.20181001.nc and map_oEC60to30v3_to_cmip6_180x360_aave.20181001.nc?

I'm not sure if I should be finding a map that's specifically "ne30pg2" or "EC30to60E2r2". 

Thank you.

**Comment by pgoddard08** [comment]
If others came across this issue. Here is another set of maps I found elsewhere in the directory tree provided in https://e3sm.org/wp-content/uploads/2020/08/e3sm-regridding_200820.pdf :
https://web.lcrc.anl.gov/public/e3sm/mapping/maps/

**Comment by jonbob** [comment]
@pgoddard08 -- please check here: https://web.lcrc.anl.gov/public/e3sm/diagnostics/maps/

**Reply by pgoddard08** [reply]
Great, thanks for your help!
